lib/Swagger/src/Configuration.php

```diff
@@ -89,7 +89,7 @@ class Configuration
      *
      * @var string
      */
-    protected $host = 'https://localhost/user-preference';
+    protected $host = 'https://localhost/';
```

Comment:
I hope not :D This was auto-generated and it's common file for all services. But it;s good to ask them :)